include/ddc/kernels/splines/spline_evaluator.hpp

```diff
@@ -208,6 +358,19 @@ class SplineEvaluator
                 });
     }
 
+    /** @brief Perform batched 1D integrations of a spline function (described by its spline coefficients) along the dimension of interest and store results on a subdomain of batch_domain.
+     *
+     * The spline coefficients represent a spline function defined on a B-splines (basis splines). They can be obtained via various methods, such as using a SplineBuilder.
```

Comment:
```suggestion
     * The spline coefficients represent a spline function defined on a B-splines (basis splines). They can be obtained via the SplineBuilder.
```
I would either give examples of the various methods either not mention it